geometry_msgs/Pose resulting_transformation
float64 transformation_peak_height
float64 persistent_transformation_peak_value
float64 rotation_peak_height # between 0 and 1
float64[9] translation_covariance
float64[3] rotation_covariance
float64 time_to_calculate # in ms


